### PR TITLE
WIP: feat(version): Include EncoreUI version in Feedback

### PR DESCRIPTION
**Pull request by CITguy**
Adding a `ENCORE_UI_VERSION` constant for use with rxFeedback

### LGTMS
- [ ] Dev LGTM
- [ ] Dev LGTM

### CODE REVIEW

**Comment by parlarjb** [comment]
Nice idea, LGTM

File: src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ angular.module('encore.ui.quarks')
         var subject = 'Encore Feedback: ' + feedback.type.label;
         var body = [
             'Current Page: ' + $location.absUrl(),
+            'EncoreUI Version: ' + ENCORE_UI_VERSION,
```
Comment (julianbonilla) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.js]: The EncoreUI version will only be reported if the Feedback Service fails.  We should also include it here: https://github.com/rackerlabs/encore-ui/blob/master/src/components/rxFeedback/scripts/rxFeedback.js#L65-L76.

Will require an update to the API side also.

**Comment by parlarjb** [comment]
Bah, can't believe I didn't remember that this would need an API update. Good catch @julianbonilla 

@CITguy I'm removing the LGTM for now, because use of the feedback service is how (theoretically) all feedback notifications now take place.

**Comment by CITguy** [comment]
Yeah, there's some other changes that need to take place as well w.r.t. grunt tasks because current implementation doesn't work for karma/unit tests.

File: src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.js (hunk 1 above)
Comment (CITguy) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.js]: Thanks. I'll take a look.

**Comment by CITguy** [comment]
I'm going to close this PR for now. It seems like there's more work than I had initially thought. I've created the following JIRA story to capture the work. https://jira.rax.io/browse/FRMW-682

**Pull request closed without merging**